### ISSUE TITLE
Automatically trigger new envoy builds when envoy releases a new version

### ISSUE THREAD

**Issue by lahabana** [kumahq/envoy-builds#72]
### Description

I'm not sure how to manage this.

Could it be dependabot?

**Comment by michaelbeaumont** [comment]
We can just check every night if there's a new Envoy release right?

**Comment by lahabana** [comment]
Yep we could trigger a 5am or so job that way when we get up we have the release ready :)

**Comment by jakubdyszkiewicz** [comment]
> Triage: the goal here is to get build out as quickly as possible so we should probably do this more often than once a day

**Comment by lukidzi** [comment]
We have a on schedule job https://github.com/kumahq/envoy-builds/actions/workflows/release-on-schedule.yaml
